Draggable.js

```diff
@@ -79,10 +80,13 @@ export default function Draggable(props) {
   );
 
   const reversePosition = React.useCallback(() => {
+    const originalOffset = {x: 0, y: 0};
+    const newOffset = typeof onReverse === "function" ? onReverse() : originalOffset;
     Animated.spring(pan.current, {
-      toValue: {x: 0, y: 0},
+      toValue: newOffset ? newOffset : originalOffset,
```

Comment:
Might be a bit more succinct as
```suggestion
      toValue: newOffset || originalOffset,
```